README.md

```diff
@@ -479,6 +479,7 @@ endpoints:
 | `alerting.email.overrides`         | List of overrides that may be prioritized over the default configuration                   | `[]`          |
 | `alerting.email.overrides[].group` | Endpoint group for which the configuration will be overridden by this configuration        | `""`          |
 | `alerting.email.overrides[].to`    | Email(s) to send the alerts to                                                             | `""`          |
+| `alerting.email.client.insecure`   | The email client does not verify the TLS certificate of the mail server                    | `false`       |
```

Comment:
```suggestion
| `alerting.email.client.insecure`   | The email client does not verify the TLS certificate of the mail server                    | `false`       |
| `alerting.email.overrides`         | List of overrides that may be prioritized over the default configuration                   | `[]`          |
| `alerting.email.overrides[].group` | Endpoint group for which the configuration will be overridden by this configuration        | `""`          |
| `alerting.email.overrides[].to`    | Email(s) to send the alerts to                                                             | `""`          |
```

Reply:
Just moving `alerting.email.client.insecure` above the overrides

Reply:
also could you update the description of `alerting.email.client.insecure` to `Whether to skip TLS verification`?

Reply:
done